desktop/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "Spectrum",
+  "description": "The community platform for the future.",
+  "version": "2.1.6",
+  "main": "src/main.js",
+  "private": true,
+  "dependencies": {
+    "electron-is-dev": "^0.3.0",
+    "electron-log": "^2.2.14",
+    "electron-updater": "^2.21.4"
+  },
+  "devDependencies": {
+    "concurrently": "^3.5.1",
+    "electron": "^1.8.4",
+    "electron-builder": "^20.8.1",
+    "wait-on": "^2.1.0"
+  },
+  "scripts": {
+    "dev": "cd ../ && concurrently \"cross-env BROWSER=none yarn dev:web\" \"wait-on http://localhost:3000 && electron ./desktop/src/main.js\"",
+    "package": "build --dir",
+    "package:mac": "build --mac",
+    "package:linux": "build --linux",
+    "package:win": "build --win --x64",
+    "package:all": "build -mwl",
+    "ship": "build -p always"
+  },
+  "build": {
+    "productName": "Spectrum",
+    "appId": "com.spectrum",
```

Comment:
This should probably be `chat.spectrum`?

Reply:
Yes you're right, it's better